### ISSUE TITLE
R-devel (to be R 4.1.0) needs a tweak in your tests:  `check.environment = FALSE`

### ISSUE THREAD

**Issue by mmaechler** [Ryo-N7/tvthemes#15]
Your package fails tests for R-devel on CRAN now.

You need changes in  `tvthemes/tests/testthat/test-thelastairbender.R` :   after line 2, insert the following _3_ lines

```r
## For R-devel (aka R 4.1.x), but also works in older R
expect_eqNe <- function(...) expect_equal(..., check.environment=FALSE)

```

and then replace  `expect_equal(`  by  `expect_eqNe(`   in these cases:

```
    128:    expect_eqNe(scale_color_avatar (palette = "FireNation"), scale_colour_avatar(palette = "FireNation"))
    145:    expect_eqNe(scale_color_avatar (palette = "WaterTribe"), scale_colour_avatar(palette = "WaterTribe"))
    162:    expect_eqNe(scale_color_avatar(palette = "EarthKingdom"), scale_colour_avatar(palette = "EarthKingdom"))
    179:  expect_eqNe(scale_color_avatarTLA(palette = "AirNomads"), scale_colour_avatarTLA(palette = "AirNomads"))
    270:  expect_eqNe(scale_color_avatar(palette = "FireNation"), scale_colour_avatar(palette = "FireNation"))
    287:  expect_eqNe(scale_color_avatar(palette = "WaterTribe"), scale_colour_avatar(palette = "WaterTribe"))
    304:  expect_eqNe(scale_color_avatar(palette = "EarthKingdom"), scale_colour_avatar(palette = "EarthKingdom"))
    321:  expect_eqNe(scale_color_avatar(palette = "AirNomads"), scale_colour_avatar(palette = "AirNomads"))
```

**Comment by mmaechler** [comment]
As a matter of fact there are other places which need a similar change,  using  `check.environment=FALSE`  or corresponding "3rd generation testthat" testing which I'm told checks the environments anyway already.

```
FAILURE (test-attackontitan.R:11:3): scale_colour_attackOnTitan equals scale_color_attackOnTitan
FAILURE (test-bighero6.R:11:3): scale_colour_bigHero6 equals scale_color_bigHero6
FAILURE (test-brooklyn99.R:65:3): scale_colour_brooklyn99/dark equals scale_color_brooklyn99/dark
FAILURE (test-brooklyn99.R:66:3): scale_colour_brooklyn99/dark equals scale_color_brooklyn99/dark
FAILURE (test-gameofthrones.R:11:3): scale_colour_stark equals scale_color_stark
FAILURE (test-gameofthrones.R:28:3): scale_colour_Lannister equals scale_color_Lannister
FAILURE (test-gameofthrones.R:45:3): scale_colour_Tyrell equals scale_color_Tyrell
FAILURE (test-gameofthrones.R:62:3): scale_colour_Targaryen equals scale_color_Targaryen
FAILURE (test-gameofthrones.R:79:3): scale_colour_Tully equals scale_color_Tully
FAILURE (test-gameofthrones.R:96:3): scale_colour_Greyjoy equals scale_color_Greyjoy
FAILURE (test-gameofthrones.R:113:3): scale_colour_Manderly equals scale_color_Manderly
FAILURE (test-gameofthrones.R:130:3): scale_colour_Stannis equals scale_color_Stannis
FAILURE (test-gameofthrones.R:147:3): scale_colour_Martell equals scale_color_Martell
FAILURE (test-gameofthrones.R:164:3): scale_colour_Arryn equals scale_color_Arryn
FAILURE (test-gravityfalls.R:11:3): scale_colour_gravityFalls equals scale_color_gravityFalls
FAILURE (test-hilda.R:67:3): scale_colour_hilda equals scale_color_hilda
FAILURE (test-hilda.R:140:3): scale_colour_hilda equals scale_color_hilda
FAILURE (test-hilda.R:213:3): scale_colour_hilda equals scale_color_hilda
FAILURE (test-kimpossible.R:11:3): scale_colour_kimPossible equals scale_color_kimPossible
FAILURE (test-parksandrec.R:177:3): scale_colour_parksAndRec equals scale_color_parksAndRec
FAILURE (test-rickandmorty.R:65:3): scale_colour_rickAndMorty equals scale_color_rickAndMorty
FAILURE (test-simpsons.R:65:3): scale_colour_simpsons equals scale_color_simpsons
FAILURE (test-spongebob.R:65:3): scale_colour_spongeBob equals scale_color_spongeBob
```

**Comment by Ryo-N7** [comment]
fixed in version 1.1.1 (released on CRAN, Dec. 14, 2020)
